src/actors/upgrade/check_output/check_html_output/check_html_output.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+import json
+import sys
+from jinja2 import Template
```

Comment:
We'll need to vendor `jinja2` with this actor I guess, can you please check jinjas' dependencies (transitively) so that we can determine what we need to vendor?

Reply:
+1. 

IMO it would be better to generate the HTML in the client side. A few advantages that I see in this approach:

1. We don't have to vendor `jinja2` with the actor.
2. In the client, we have the option to add a new output types in a decoupled way, i.e., we don't need another actor to generate a XML or CSV report.
3. If we want to make refinements to the web client (Cockpit), we don't have to change the actor as well.

One disadvantage for this approach is that each client (e.g.., `leappctl` or `cockpit-ui`) needs its own report template. I think this is acceptable, though.

If we decide to go with this approach, I think it would be OK to work on that  in a separate PR.

Reply:
> One disadvantage for this approach is that each client (e.g.., leappctl or cockpit-ui) needs its own report template. I think this is acceptable, though.

I guess you mean templates on the client side and report actor will produce report in suitable format for another processing (e.g. json).

Reply:
Yup, that's right.